packages/design-system-react-native/src/components/BadgeStatus/README.md

```diff
@@ -0,0 +1,147 @@
+# BadgeStatus
+
+The `BadgeStatus` component indicates the status an entity is on. It is a small circular badge used to indicate status, such as activity, new items, or alerts. It supports different statuses, sizes, and an optional border for better visibility.
```

Comment:
Suggestion: Let's match the component description with Figma, as it’s a low-hanging fruit that helps with alignment across platforms.  

```suggestion
`BadgeStatus` indicates the status an entity is on.
```

Reply:
The description I had includes the component description with FIgma with additional information. It's an enhancement